packages/terra-core-docs/src/terra-dev-site/doc/hyperlink/example/HyperlinkButton/AudioHyperlinkButton.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import HyperlinkButton from 'terra-hyperlink/lib/HyperlinkButton';
+
+export default () => <HyperlinkButton onClick={() => {}} variant="audio" text="Audio Hyperlink Button" />;
```

Comment:
Is the onClick prop really needed in these examples since it doesn't do anything?

Reply:
It is marked as required so added it